libmscore/textbase.cpp

```diff
@@ -982,6 +1005,24 @@ void TextBlock::layout(TextBase* t)
       _bbox.translate(rx, 0.0);
       }
 
+//---------------------------------------------------------
+//   fragmentsWithoutEmpty
+//---------------------------------------------------------
+
+QList<TextFragment>* TextBlock::fragmentsWithoutEmpty()
+      {
+      QList<TextFragment>* list = new QList<TextFragment>();
+      for (auto x :_fragments) {
+            if (x.text.isEmpty()) {
+                  continue;
+                  }
```

Comment:
these braces seem redundant

Reply:
Also a simple

    if (!x.text.isEmpty())
         list->append(x);

Should be sufficient? No `else`, no `continue` needed